tox.ini

```diff
@@ -90,21 +106,21 @@ max-line-length = 99
 ignore = E203, W503
 
 [coverage:run]
-omit = tox/__main__.py
+branch = false
 
 [coverage:report]
 skip_covered = True
 show_missing = True
 exclude_lines = if __name__ == ["']__main__["']:
 
 [coverage:paths]
-source = tox
-         {toxworkdir}/*/lib/python*/site-packages/tox
-         {toxworkdir}/*/Lib/site-packages/tox
-         {toxworkdir}/pypy*/site-packages/tox
+source = src/tox
+         .tox/*/lib/python*/site-packages/tox
+         .tox/*/Lib/site-packages/tox
+         .tox/pypy*/site-packages/tox
```

Comment:
this is by far my least favorite part of moving to a `src` directory, the coverage reports become unreadable :(

we've used [this script](https://github.com/Yelp/venv-update/blob/master/tests/testing/fix_coverage.py) or a derivative of it to try and make this more sane (it combines the coverage data from site-packages over top the local directory)